ci/travis/run_build.sh

```diff
@@ -42,15 +42,15 @@ build_doxygen() {
     cd ../..
 
     #Generate *.dox files for drivers and projects
-    ${TOP_DIR}/ci/travis/gen_dox.sh
+    . ${TOP_DIR}/ci/travis/gen_dox.sh
 
     # Build Documentation
     cd ${TOP_DIR}/doc
     mkdir -p build && cd build
     cmake ..
     cd ../..
 
-    ./ci/travis/doxygen.sh
+    . ./ci/travis/doxygen.sh
```

Comment:
Is ". ." correct?

Reply:
I can use `source` instead of the first dot if it is more intelligible: https://ss64.com/bash/source.html

I wanted to keep the same approach as for the rest of the builds:
https://github.com/analogdevicesinc/no-OS/blob/789dee35a8967632a312609733d3568218cfb9a0/ci/travis/run_build.sh#L17

https://github.com/analogdevicesinc/no-OS/blob/789dee35a8967632a312609733d3568218cfb9a0/ci/travis/run_build.sh#L13

This is required, as specified in the commit description, to propagate the `TOP_DIR` variable inside the doxygen scripts.

Reply:
OK, thanks for the details.